common/questions.ts

```diff
@@ -94,6 +94,27 @@ export const Questions: Array<QuestionDefinition> = [
     ['Freshman', 'Sophomore', 'Junior', 'Senior', 'Other'],
     true
   ),
+  makeDropdown(
+    'What school do you attend?',
+    [
```

Comment:
I told Karen it was probably fine to just do Boston schools since we're going to be in person, but good point. Do you know if we're planning on accepting all applicants? I'll also ask Cari/leadership